src/System.Private.ServiceModel/tests/Common/Infrastructure/testproperties.props

```diff
@@ -150,6 +150,10 @@
     <PropertyGroup Condition="'$(SSL_Available)' == ''">
       <SSL_Available></SSL_Available>
     </PropertyGroup>
+
+  <PropertyGroup Condition="$(WithCategories.Contains('failing'))">
```

Comment:
@iamjasonp -- I made the Contains case-insensitive using the pattern used by other targets files.
@zhenlan -- regarding when the condition will be true -- the syntax used by other repo's (and by one flavor of our former nightly runs) is to say /p:WithCategories="OuterLoop;failing" when you want to include tests marked with [ActiveIssue].  These semantics are part of build tools.  Now that we have moved to [Issue], I thought it would be reasonable to allow the build-tools syntax to continue to be used.